pydir/wcsfit.cpp

```diff
@@ -51,6 +51,7 @@ py::array_t<T> vectorToNumpy(std::vector<std::vector<T>> const &vect) {
 PYBIND11_MODULE(wcsfit, m) {
     m.attr("DEGREE") = DEGREE;
     m.attr("ARCSEC") = ARCSEC;
+    m.attr("EclipticInclination") = EclipticInclination;
```

Comment:
As I noted in the other review, this is something you could get from astropy instead, I'd hope?

Reply:
`astropy` has a few options for calculating the ecliptic, none of which exactly match the value that is hard-coded in `gbdes`. I want to replicate what is in `gbdes` as closely as possible, which is why I want to pull out this variable.